test/models/job_execution_test.rb

```diff
@@ -123,6 +126,16 @@
     lines.must_include "DEPLOYER_NAME=John Doe"
     lines.must_include "REVISION=master"
     lines.must_include "TAG=v1"
+    job.output.wont_include "DEPLOY_GROUPS"
+  end
+
+  it 'exports deploy_group information if deploy groups present' do
+    job.update(command: 'env | sort')
+    job.deploy = deploy
+    execute_job
+    lines = job.output.split "\n"
+    lines.must_include "# Deploy URL: #{deploy.url}"
+    lines.must_include 'DEPLOY_GROUPS=\;\|sudo\ make-sandwich\ / Pod1 Pod2'
```

Comment:
a `it "does not add deploy group if deploy groups are missing"` would be nice :)